bundles/af-core/src/test/resources/form/formcontainer/exporter-submissionViewWithSubmissionAttribute.json

```diff
@@ -0,0 +1,73 @@
+{
+    "id": "L2NvbnRlbnQvZm9ybXMvYWYvZGVtbw==",
+    "fieldType": "form",
+    "lang": "en",
+    "title": "afv2",
+    "action": "https://forms.adobe.com/adobe/forms/af/submit/L2NvbnRlbnQvZm9ybXMvYWYvZGVtby9qY3I6Y29udGVudC9mb3JtY29udGFpbmVyLXN1Ym1pc3Npb25WaWV3Lm1vZGVsLmpzb24=",
+    "properties": {
+        "themeRef": "/libs/fd/af/themes/canvas",
+        "schemaType": "none",
+        "useExternalEmailTemplate": "true",
+        "fd:path": "/content/forms/af/demo/jcr:content/formcontainer-submissionView",
+        "fd:schemaType": "BASIC",
+        "fd:isHamburgerMenuEnabled": false,
+        "fd:roleAttribute": null,
+        "fd:formDataEnabled": false,
+        "fd:customFunctionsUrl": "/adobe/forms/af/customfunctions/L2NvbnRlbnQvZm9ybXMvYWYvZGVtbw==",
+        "fd:dataUrl": "/adobe/forms/af/data/L2NvbnRlbnQvZm9ybXMvYWYvZGVtbw=="
+    },
+    "events": {
+        "custom:setProperty": [
+            "$event.payload"
+        ]
+    },
+    ":itemsOrder": [
+        "textinput"
+    ],
+    "metadata": {
+        "grammar": "json-formula-1.0.0",
+        "version": "1.0.0"
+    },
+    "adaptiveform": "0.14.2",
+    "fd:submit": {
```

Comment:
fd:submit should be inside properties, it should not be at top level

Reply:
made the change